HHS/OPRE-OPS
# feat: ✨ adds eslint-plugin-jsx-a11y

## What changed

-  adds eslint-plugin-jsx-a11y and corrects a11y issues

## Issue

#1249 

## How to test

- automated tests pass
- could also add this component and see if the linter gives you errors in your IDE.
```jsx
function TestA11y() {
  return (
    <div>
      <img src="example.jpg" />
      <div onClick={() => console.log('clicked')} role="button">Click me</div>
      <a href="#">Learn more</a>
    </div>
  );
}

export default TestA11y;
```

## Definition of Done Checklist
